include/triton/Conversion/TritonGPUToLLVM/Utility.h

```diff
@@ -368,7 +369,7 @@ inline bool isKernel(FunctionOpInterface funcOp) {
 inline Value getStackPointer(RewriterBase &rewriter,
                              FunctionOpInterface funcOp) {
   if (!isKernel(funcOp)) {
-    return funcOp.getArgument(funcOp.getNumArguments() - 1);
+    return funcOp.getArgument(funcOp.getNumArguments() - 2);
```

Comment:
We may need to document it more explicitly: what is `funcOp.getNumArguments() - 2` and what is `funcOp.getNumArguments() - 1`